jobbergate-api/tests/apps/job_scripts/test_services.py

```diff
@@ -209,17 +234,38 @@ def time_now(self):
             yield time_now
 
     @pytest.fixture
-    async def dummy_data(self, fill_job_script_data, time_now, synth_services):
+    async def dummy_data(
+        self, fill_job_script_data, fill_job_submission_data, time_now, synth_services
+    ) -> dict[TestEntryInfo, dict[str, Any]]:
         """
         Create dummy test data.
         """
-        result = []
-        for i, is_archived in product([0, 2, 4], [True, False]):
-            data = fill_job_script_data(name=f"name-{i}", is_archived=is_archived)
-            with pendulum.test(time_now.add(days=i)):
+        result = {}
+        for entry_info in (
+            TestEntryInfo(*p)
+            for p in product(
+                [0, 1, 2],
+                [None, 0, 1, 2, 3],
+                [True, False],
```

Comment:
These values should probably be labeled somehow to make this more readable. (Comment maybe?)

Reply:
Fair enough, I named them properly for clarification: [0c7f836](https://github.com/omnivector-solutions/jobbergate/pull/563/commits/0c7f8363e6cec4284b7c19c9f3119430c0f2df89)